src/lime/_internal/backend/flash/FlashAudioSource.hx

```diff
@@ -117,6 +117,16 @@ class FlashAudioSource
 		return loops = value;
 	}
 
+	public function getPitch():Float
+	{
+		return 1;
+	}
+
+	public function setPitch(value:Float):Float
+	{
```

Comment:
Maybe you should add a trace thing like
`haxe.Log.trace("Pitch not Supported on Flash.", cast {fileName: "AudioSource"});`
It's my understanding lol

Reply:
If anything, we'd probably use `lime.utils.Log.debug()` so as to exclude the message from release builds.

Reply:
> If anything, we'd probably use `lime.utils.Log.debug()` so as to exclude the message from release builds.

Ok, How about `lime.utils.Log.debug("Pitch not Supported on Flash.", cast {className: "AudioSource"});`?

Reply:
Lime doesn't use `Log.debug()` much, so I went with `verbose()` instead.